docs/guides/latest/advance-use-case/restore-in-pvc-template.md

```diff
@@ -0,0 +1,831 @@
+
+Run the following command to watch RestoreSession phase,
+
+```console
+$ watch -n 3 kubectl get restoresession -n demo
+Every 3.0s: kubectl get restoresession -n demo               suaas-appscode: Mon Jul  8 18:39:58 2019
+
+NAME                 REPOSITORY-NAME   PHASE       AGE
+restore-deployment   gcs-repo          Succeeded   3m27s
+```
+
+So, we can see from the output of the above command that the restore process succeeded.
+
+**Verify Restored PVC:**
+
+Once a restore process is complete, we will see that new PVCs with the name `restore-data` and `restore-config` have been created.
+
+Verify that the PVCs have been created by the following command,
+
+```console
+$ kubectl get pvc -n demo
+NAME             STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+restore-config   Bound    pvc-6aab94dc-10b2-4c36-8768-89b20a7a24ed   2Gi        RWO            standard       32s
+restore-data     Bound    pvc-8296da99-b813-466a-b9f2-efff1faeee17   2Gi        RWO            standard       32s
+```
+
+Notice the STATUS field. `Bound` indicates that PVCs have been initialized from the respective backed up volumes.
+
+**Verify Restored Data:**
+
+We will create a new Deployment with the restored PVCs to verify whether the backed up data have been restored.
+
+Below is the YAML of the Deployment that we are going to create,
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: stash-demo
+  name: restore-demo
+  namespace: demo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: stash-demo
+  template:
+    metadata:
+      labels:
+        app: stash-demo
+      name: busybox
+    spec:
+      containers:
+      - args:
+        - sleep
+        - "3600"
+        image: busybox
+        imagePullPolicy: IfNotPresent
+        name: busybox
+        volumeMounts:
+        - mountPath: /restore/data
+          name: restore-data
+        - mountPath: /source/config
+          name: restore-config
+      restartPolicy: Always
+      volumes:
+      - name: restore-data
+        persistentVolumeClaim:
+          claimName: restore-data
+      - name: restore-config
+        persistentVolumeClaim:
+          claimName: restore-config
+```
+
+Let's create the deployment we have shown above.
+
+```console
+$ kubectl apply -f ./docs/examples/guides/latest/advance-use-case/restore-in-pvc-template/deployment/restore-deployment.yaml
+deployment.apps/restore-demo created
+```
+
+Now, wait for pod of the deployment to go into the `Running` state.
+
+```console
+$ kubectl get pod -n demo
+NAME                                READY   STATUS    RESTARTS   AGE
+restore-demo-85fbcb5dcf-vpbt8       1/1     Running   0          2m50s
+```
+
+Verify that the sample data has been created in `/source/data` and `/source/config` directory using the following command,
```

Comment:
Verify that the sample data has been created -> Verify that the backed up data has been restored